views/users/add.html

```diff
@@ -5,12 +5,16 @@
 {% endblock %}
 
 {% block header %}
-  {{ super() }}
+  {% if setup %}
+    {% include "_header-not-auth.html" %}
+  {% else %}
+    {{ super() }}
+  {% endif %}
 {% endblock %}
 
 {% block main %}
   <div class="introduction">
-    <h1 class="introduction--h1">{{config.actions.add|capitalize}}</h1>
+    <h1 class="introduction--h1">{{title|capitalize}}</h1>
```

Comment:
This filter is transforming `CMS` in to `cms`. While it may be needed for some generic use cases, it's causing weirdness with this case